ktx2Decoder/src/transcoderManager.ts

```diff
@@ -0,0 +1,44 @@
+import { transcodeTarget, sourceTextureFormat, Transcoder } from './transcoder';
+import { Nullable } from './types';
+import { WASMMemoryManager } from './wasmMemoryManager';
+
+/**
+ * @hidden
+ */
+export class TranscoderManager {
+
+    public static _Transcoders: Array<typeof Transcoder> = [];

```

Comment:
Why do you need `typeof`?

Reply:
Because I use a type to create an instance with:
```
transcoder = new TranscoderManager._Transcoders[i]();
```
I can't add an already created instance in `_Transcoders` because I need to create new instances for each thread in which the `TranscoderManager` is instanciated.

Reply:
Ah, I was looking for that and somehow missed it.